resources/.gitignore

```diff
@@ -0,0 +1,4 @@
+connectd
+demuxer
+muxer
+remoteit
```

Comment:
I'd rather keep all the .gitignore stuff in the single file if possible please  :)